subcommand/controller/command.go

```diff
@@ -204,11 +210,20 @@ func (c *Command) upsertConsulResources(consulClient *api.Client, ecsMeta awsuti
 		)
 	}
 
-	serviceBindingRule := &api.ACLBindingRule{
-		Description: "Bind a service identity from IAM role tag for ECS",
-		AuthMethod:  serviceAuthMethod.Name,
-		BindType:    api.BindingRuleBindTypeService,
-		BindName:    fmt.Sprintf(`${entity_tags.%s}`, authMethodServiceNameTag),
+	bindingRules := []*api.ACLBindingRule{
+		{
+			Description: "Bind a service identity from IAM role tag for ECS",
+			AuthMethod:  serviceAuthMethod.Name,
+			BindType:    api.BindingRuleBindTypeService,
```

Comment:
I'd suggest also using the bindtype templated-policy here, the goal is to replace the `BindingRuleBindTypeService` eventually with `BindingRuleBindTypeTemplatedPolicy` 

```
BindType: api.BindingRuleBindTypeTemplatedPolicy,
BindName: "builtin/service",
BindVars: &{
    Name: fmt.Sprintf(`${entity_tags.%s}`, authMethodServiceNameTag),
},
```

Reply:
Yes. That is the north star. Have captured the same in the PR description too. Once Consul core supports it, I can remove the logic to create roles and policies in ECS controller and use the builtin policy.

Reply:
yes i understand that, binding rules already support the suggestion here

Reply:
Oops. I misread your comment. I thought it was for the API gateway example. Will modify this as well

Reply:
This will be taken up as part of a followup PR